testdata/p4_16_samples_outputs/pna-too-big-label-name-dpdk.p4.spec

```diff
@@ -0,0 +1,91 @@
+
+struct ethernet_t {
+	bit<48> dstAddr
+	bit<48> srcAddr
+	bit<16> etherType
+}
+
+;oldname:ipv4_ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd_t
+struct ipv4_dddddddddddddddddddddddd0 {
+	bit<8> version_ihl
+	bit<8> diffserv
+	bit<16> totalLen
+	bit<16> identification
+	bit<16> flags_fragOffset
+	bit<8> ttl
+	bit<8> protocol
+	bit<16> hdrChecksum
+	bit<32> srcAddr
+	bit<32> dstAddr
+}
+
+struct next_hop_arg_t {
+	bit<32> vport
+}
+
+struct main_metadata_t {
+	bit<32> pna_main_input_metadata_direction
+	bit<32> pna_main_input_metadata_input_port
+	bit<32> pna_main_output_metadata_output_port
+	bit<32> MainControlT_tmpDir
+}
+metadata instanceof main_metadata_t
+
+header ethernet instanceof ethernet_t
+header ipv4 instanceof ipv4_dddddddddddddddddddddddd0
```

Comment:
please see here header type modified.

Reply:
So is this addressing the other issue in header instance as well?

Reply:
no